content/source/docs/enterprise/api/organizations.html.md

```diff
@@ -0,0 +1,332 @@
+---
+layout: enterprise2
+page_title: "Organizations - API Docs - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-api-organizations"
+---
+
+# Organizations API
+
+-> **Note**: These API endpoints are in beta and are subject to change.
+
+The Organizations API is used to list, show, create, update, and destroy organizations.
+
+## List Organizations
+
+`GET /organizations`
+
+Status  | Response                                        | Reason
+--------|-------------------------------------------------|----------
+[200][] | [JSON API document][] (`type: "organizations"`) | The request was successful
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+[JSON API document]: https://www.terraform.io/docs/enterprise/api/index.html#json-api-documents
+
+### Sample Request
+
+```shell
+curl \
+  --header "Authorization: Bearer $ATLAS_TOKEN" \
+  --header "Content-Type: application/vnd.api+json" \
+  --request GET \
+  https://app.terraform.io/api/v2/organizations
+```
+
+### Sample Response
+
+```json
+{
+  "data": [
+    {
+      "id": "hashicorp",
+      "type": "organizations",
+      "attributes": {
+        "name": "hashicorp",
+        "created-at": "2017-09-07T14:34:40.492Z",
+        "email": "user@example.com",
+        "session-timeout": null,
+        "session-remember": null,
+        "collaborator-auth-policy": "password",
+        "enterprise-plan": "pro",
+        "permissions": {
+          "can-update": true,
+          "can-destroy": true,
+          "can-create-team": true,
+          "can-create-workspace": true,
+          "can-update-oauth": true,
+          "can-update-api-token": true,
+          "can-update-sentinel": true,
+          "can-traverse": true,
+          "can-create-workspace-migration": true
+        }
+      },
+      "links": {
+        "self": "/api/v2/organizations/hashicorp"
+      }
+    }
+  ]
+}
+```
+
+## Show an Organization
+
+`GET /organizations/:organization_name`
+
+Parameter            | Description
+---------------------|------------
+`:organization_name` | The name of the organization to show
+
+Status  | Response                                        | Reason
+--------|-------------------------------------------------|----------
+[200][] | [JSON API document][] (`type: "organizations"`) | The request was successful
```

Comment:
- What happens if the organization doesn't exist? 404?
- Are some users blocked from seeing some organizations (404?), or can any user see any org?